test/project_types/script/commands/test_test.rb

```diff
@@ -29,6 +29,13 @@ def test_calls_test_service
         perform_command
       end
 
+      def test_help
+        ShopifyCli::Context
+          .expects(:message)
+          .with('script.test.help', ShopifyCli::TOOL_NAME)
+        Script::Commands::Test.help
+      end
+
```

Comment:
missing a test for extended_help?

Reply:
`test` has no options, so I left the method `test.extended_help` and the test for it out. The concern I have about including this method with nothing to put in extended help is that by default it gets this extra newline (second command in screenshot), so I thought it was better to add this in later if there are options rather than now with some formatting.

<img width="449" alt="Screen Shot 2020-06-09 at 7 56 46 AM" src="https://user-images.githubusercontent.com/64974039/84163903-d2c7a580-aa26-11ea-85ea-0c4b446639e1.png">

Reply:
Ah, right.  Besides, we are deleting `test` command soon anyway.  👍